modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,35 +6,61 @@
 For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
 
-[id="upgrading-openshift_{context}"]
-== Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+== Updating from {product-title} 4.13 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.13 to 4.14, first update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic migration is enabled by default.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by performing an administrator acknowledgment:
+
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.13-kube-127-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
 
 [IMPORTANT]
 ====
-Carefully review the following consequences before opting in to migration:
+If you do *not* update to vSphere 7.0 Update 3L or 8.0 Update 2 and use an administrator acknowledgment to update to {product-title} 4.14, known issues can occur. link:https://access.redhat.com/node/7011683[Before proceeding with the administrator acknowledgement, carefully read this knowledge base article].
+====
 
-* link:https://access.redhat.com/node/7011683[There are known issues that can occur that can cause the migration to fail].
+== Updating from {product-title} 4.12 to 4.13
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.13, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic CSI migration for vSphere only occurs if you opt in.
 
-* Enabling migration cannot be undone.
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by:
 
-* Migration can take awhile to complete depending on how many nodes are on the cluster.
+* Initiating the update from a {product-title} 4.12 Z-stream.
+
+* Performing an administrator acknowledgment by running the following command:
++
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-126-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
 
-* Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
+[IMPORTANT]
+====
+If you update to {product-title} 4.13 without updating vSphere, and then opt in to migration, known issues can occur. Review the consequences listed above for migration before opting in and link:https://access.redhat.com/node/7011683[carefully read this knowledge base article].
```

Comment:
add link to the opt in doc ? https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/persistent-storage-csi-migration.html#upgrading-openshift_persistent-storage-csi-migration

Reply:
Done.